packages/web/src/pages/chat-page/components/ChatHeader.module.css

```diff
@@ -1,7 +1,7 @@
 .root {
   display: flex;
   height: 112px;
-  margin-left: -15px;
+  margin-left: -20px;
```

Comment:
Talked about using a CSS var or something for this offline, other than that this is good to go!